data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you both should head back to the ship and pick her and Kirrie back up. You two meet up with the rest of the group.`
+				accept
+			label holeintheground
+			`	You two carefully make your way down the hole, down to the bottom, around ten meters below the flat area with the barn.`
+			`	As you were coming down, Tugroob pointed to the cracked, rocky walls, saying that most of them were unstable, and you should watch out for rockslides.`
+			`	Looking around, you two spot several smaller holes on the walls, many being just the right size for a human or Saryd arm to squeeze in, and a few larger cracks, that you two shouldn't have much trouble going through.`
+			choice
+				`	(Reach your arms inside the smaller holes to look for anything there.)`
+					goto rockyarm
+				`	(Go for the larger cracks and see if they lead to anything.)`
+					goto rockraider
+			label rockyarm
+			`	You carefully go about putting your arm inside a few of the holes, with Tugroob helping you up for the ones you couldn't reach, but ultimately you find nothing.`
+			`	A few small rocks fell as you went about the whole thing, but none hit neither of you, thankfully. Up above, the storm has already started to bring in heavy amounts of sand inside.`
+			`	"To remain here for long, dangerous, will be, Captain," Tugroob says.`
+			choice
+				`	(Head for the larger cracks in the wall now.)`
+					goto rockraider
+				`	(Head back up while you still can, and check the barn.)`
+					goto barnyedwards
+			label barnyedwards
+			`	Tugroob crawls back up while you cling to them, as the path you took before proves to be much too risky with the reduced visibility.`
+			`	You two head inside the barn, to find the place nearly entirely empty, save for a few sets of crates and barrels around the corners.`
+			`	You follow Tugroob to the upper levels, but once again find nothing but dusty containers. A small gap between the planks here allows some sand to come inside, the storm is getting closer.`
+			`	"Much too obvious a hiding spot, this would be," they say, but seeing as the storm doesn't go down, you two agree to break open the crates, just in case. Regrettably, you find nothing.`
+			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you both should head back to the ship and pick her and Kirrie back up. You two meet up with the rest of the group.`
+				accept
+			label rockraider
+			apply
+				"resistance evidence" ++
+			`	After finding a hole that both of you can go through, you two head on inside, Tugroob lighting up the path below with a flashlight. A couple larger rocks occasionaly fall, and Tugroob has to prevent some from hitting you, quickly crawling around the walls to try and slow them down to give you time to move away.`
```

Comment:
```suggestion
			`	You find a hole large enough for both of you and enter, Tugroob holding a flashlight to illuminate the path below. Larger rocks occasionally fall, and Tugroob has to prevent some from hitting you, quickly crawling around the walls to try and slow them down so you have time to move away.`
```